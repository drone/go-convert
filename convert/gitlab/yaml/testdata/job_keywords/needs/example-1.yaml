# https://docs.gitlab.com/ee/ci/yaml/#needs

linux:build:
  stage: build
  script: echo "Building linux..."

mac:build:
  stage: build
  script: echo "Building mac..."

lint:
  stage: test
  needs: []
  script: echo "Linting..."

linux:rspec:
  stage: test
  needs: ["linux:build"]
  script: echo "Running rspec on linux..."

mac:rspec:
  stage: test
  needs: ["mac:build"]
  script: echo "Running rspec on mac..."

production:
  stage: deploy
  script: echo "Running production..."
  environment: production