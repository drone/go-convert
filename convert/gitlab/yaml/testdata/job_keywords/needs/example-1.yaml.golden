lint:
    needs: []
    script:
        - echo "Linting..."
    stage: test
linux:build:
    script:
        - echo "Building linux..."
    stage: build
linux:rspec:
    needs:
        - job: linux:build
    script:
        - echo "Running rspec on linux..."
    stage: test
mac:build:
    script:
        - echo "Building mac..."
    stage: build
mac:rspec:
    needs:
        - job: mac:build
    script:
        - echo "Running rspec on mac..."
    stage: test
production:
    environment:
        name: production
    script:
        - echo "Running production..."
    stage: deploy
